### PR TITLE
Appdata for next release

### PR DESCRIPTION
**Pull request by jeremypw**
Adds appdata for the next release and syncs with master.

### CODE REVIEW

**Review by cassidyjames — CHANGES_REQUESTED** [review]
6.0 notes are already there; you've duplicated them in this PR.

**Comment by jeremypw** [comment]
Oops - not sure how that happened!

File: data/io.elementary.files.appdata.xml.in.in [hunk 1]
```diff
@@ -56,6 +56,23 @@
...
+        <ul>
+          <li>Fix truncation of slot if reclicked in Column View</li>
+        </ul>
+        <p>Improvements:</p>
+        <ul>
+          <li>Show more keyboard accelerators in menus</li>
+        </ul>
+        <p>Other updates:</p>
+        <ul>
+          <li>Now requires granite version >= 6.1.0</li>
```
Comment (danrabbit) [inline data/io.elementary.files.appdata.xml.in.in:71]: I'm not sure that this is a user-facing change. Is there a user-facing benefit about this we can list?

File: data/io.elementary.files.appdata.xml.in.in [hunk 2]
```diff
@@ -56,6 +56,23 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="6.0.1" date="2021-07-15" urgency="medium">
+      <description>
+        <p>Fixes:</p>
+        <ul>
+          <li>Fix truncation of slot if reclicked in Column View</li>
```
Comment (danrabbit) [inline data/io.elementary.files.appdata.xml.in.in:63]: What does "slot" mean in this context?

**Review by danrabbit — CHANGES_REQUESTED** [review]
Way to be proactive! Just a couple of questions

File: data/io.elementary.files.appdata.xml.in.in (hunk 1 above)
Comment (jeremypw) [reply data/io.elementary.files.appdata.xml.in.in:71]: The benefit at the moment is the ability to put keyboard accelerators in the tab context menus - which is already covered elsewhere.  I'll remove it.  I guess any user who wants to compile from source will find this out from the README file.

File: data/io.elementary.files.appdata.xml.in.in (hunk 2 above)
Comment (jeremypw) [reply data/io.elementary.files.appdata.xml.in.in:63]: I guess "column" would be a better word.  A slot in the context of Files is the widget that displays a directory listing.

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**